browser/pages/confirm/controller.js

```diff
@@ -71,6 +72,10 @@ class ConfirmController {
       ).catch(
         ()=>this.setIsDisabled()
       );
+      Platform.isVirtualizationEnabled().then(status => {
```

Comment:
Almost there. Now detection for installed components and for virtualization would run at the same time, but update on angular ui would happen twice. One in this.setIsDisabled() and second one in lambda after virtualization detector is finished. Lets move it to line 62 and add to detectors array first.
``` JavaScript
detectors.push(Platform.isVirtualizationEnabled().then(status => {
 $scope.virtualization = status;
});
```

$scope.apply() call is not required, because it is going to be done inside `this.setIsEnabled()`.